frontend/src/component/GuestPage/UploadForm.jsx

```diff
@@ -77,7 +77,7 @@ class UploadForm extends React.Component {
                 target='popoverPasswordInfo'
                 // INFO - CH - 20200507 - ignoring rule react/jsx-handler-names for prop bellow because it comes from external lib
                 toggle={this.handleTogglePopoverPasswordInfo} // eslint-disable-line react/jsx-handler-names
-                trigger={isMobile ? 'focus' : 'hover'}
+                trigger={isMobile ? 'click' : 'hover'}
```

Comment:
As the test & trigger values are always the same, wouldn't a `TracimPopover` component be appropriate to factorize this behavior? Or a `getPopoverTrigger()` function?